### DISCUSSION TITLE
visionOS compatibility

### DISCUSSION THREAD

**Discussion by walkerzack** [simonoliver/UnityFigmaBridge#46] [General]
Hi there!

I am working on a small application for the Apple Vision Pro and wanted to use Unity Figma Bridge to simplify my UI creation workflow. 
In general, the setup was super easy and straight forward, but ran into this issue with shaders in visionOS / PolySpatial:

https://discussions.unity.com/t/non-shader-graph-shader-not-supported-or-materialx-encoding-missing/1503931/2

I have no idea how much effort it would be to update the shaders to fix this issue, but do you have any plans to support PolySpatial in the future?

Thanks in advance!

**Comment by simonoliver** [comment]
Hey Zack,

I'm not super familiar with PolySpatial but it should be possible to recreate the FigmaImageShader in ShaderGraph (though as there's a fair bit of complexity there it would take some time). I'm not able to invest a lot of time in my open source packages at the moment but feel free to open an issue for an enhancement in case myself or someone else is able to do this work.

Simon

**Reply by walkerzack** [reply]
Hi Simon,

thanks for the quick reply!
I added a new feature request as you suggested :)
